_posts/2015-09-04-channel.html

```diff
@@ -2955,6 +2955,234 @@
 	...
   }</code></pre>
 
+		<h3>Video trim</h3>
+		<p>Can set video playback start and end point in seconds.</p>
```

Comment:
"You can cut your videos by setting the playback start and end positions. When watching a trimmed video, only the chosen interval is shown in the video player." vagy valami hasonlóra gondolnék.

Szerintem érdemes lenne arról is tájékoztatni a felhasználókat, hogy "Trimming does not affect the original video file in any way: even though a video has been trimmed you can still download the whole video."